charts/identity-api/values.yaml

```diff
@@ -50,6 +50,7 @@ env:
   DEV_LICENSE_ADDR: '0xdb6c0dBbaf48b9D9Bcf5ca3C45cFF3811D70eD96'
   STAKING_ADDR: '0xA010a42DE263592E631d3DCDC621ae7BFfCca338'
   CONNECTION_ADDR: '0x41799E9Dc893722844E771a1C1cAf3BBc2876132'
```

Comment:
once we merge, it will be live only on dev? For prod we will do separate PR later?

Reply:
My hope was that you guys could resolve any questions like, e.g., the `"uri"` naming, before we committed to something on prod.

Reply:
ok, sounds reasonable to me